packages/zowe-explorer/__tests__/__unit__/extension.unit.test.ts

```diff
@@ -387,6 +389,7 @@ async function createGlobalMocks() {
         value: globalMocks.mockPromptUserWithNoConfigs,
         configurable: true,
     });
+    Object.defineProperty(vscode.window, "registerWebviewPanelSerializer", { value: jest.fn() });
```

Comment:
Please define a stub for this function in the module mock for VS Code (rather than using `Object.defineProperty`) to prevent impacting future tests.

The module mock is located in `packages/zowe-explorer/__tests__/__mocks__/vscode.ts`